public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
한글 앱이면 lang ko로 변경해주면 좋을 것 같네요
```suggestion
<html lang="ko">
```
+ application에 맞는 타이틀로 수정해보시면 좋을 것 같네요